taps/tapManager.go

```diff
@@ -124,7 +124,7 @@ func ConfigIPtables(tapName, hostIface string) error {
 		}
 	}
 	cmd := exec.Command(
-		"sudo", "iptables", "-t", "nat", "-A", "POSTROUTING", "-o", hostIface, "-j", "MASQUERADE",
+		"sudo", "iptables", "--wait", "-t", "nat", "-A", "POSTROUTING", "-o", hostIface, "-j", "MASQUERADE",
```

Comment:
I have only seen this so far for MASQUERADE. Not sure if the other needs it as well.

Reply:
I see also for the other commands now.